circle.yml

```diff
@@ -17,7 +18,7 @@ machine:
 dependencies:
   cache_directories:
     - "addon/node_modules"
-    - "/home/ubuntu/firefox-downloads"
+    - # "/home/ubuntu/firefox-downloads"
```

Comment:
I can remove this back to how it was. Nightly had a bug last week that caused it to not read the capabilities given to it by geckodriver.